docs/Makefile

```diff
@@ -9,12 +9,16 @@ SOURCEDIR     = .
 BUILDDIR      = _build
 
 # Put it first so that "make" without argument is like "make help".
-help:
+help:	fledge_plugins.rst
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
 .PHONY: help Makefile
 
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
-%: Makefile
-	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
\ No newline at end of file
+%: Makefile fledge_plugins.rst
+	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+fledge_plugins.rst: 
```

Comment:
To avoid conflict Better to rename the target without extension, as we are creating same output file with same name

Reply:
Sorry, I don't understand your comment here?

Reply:
As `rst` output is being saved & named as "fledge_plugins.rst".
So, If the file exists  `make` actually got confused. As by default Makefile targets are the file targets.
Hence either to change the target/output file name OR add this target in `.PHONY`

Reply:
Make is not at all confused by this. However it does no harm to add to .PHONY, although of course this is a file that is being created so is a very normal way to use make.